### PR TITLE
Changed the long inputs in ScheduledInterview DAOs to Instants.

### PR DESCRIPTION
**Pull request by GrantComm**
Made this change for consistency across project. See pr #108

### CODE REVIEW

File: src/main/java/com/google/sps/data/DatastoreScheduledInterviewDao.java [hunk 1]
```diff
@@ -156,13 +157,18 @@ public Entity scheduledInterviewToEntityForUpdate(ScheduledInterview scheduledIn
...
     Filter minFilter =
-        new FilterPredicate("startTime", FilterOperator.GREATER_THAN_OR_EQUAL, minTime);
+        new FilterPredicate(
+            "startTime", FilterOperator.GREATER_THAN_OR_EQUAL, minTime.toEpochMilli());
     Filter maxFilter =
-        new FilterPredicate("startTime", FilterOperator.LESS_THAN_OR_EQUAL, (maxTime - 3600000));
+        new FilterPredicate(
+            "startTime",
+            FilterOperator.LESS_THAN_OR_EQUAL,
+            maxTime.minus(60, ChronoUnit.MINUTES).toEpochMilli());
```
Comment (meagancheese) [inline src/main/java/com/google/sps/data/DatastoreScheduledInterviewDao.java:176]: Make sure the comment is here as well. I know it gets rough when it's in one PR and not another but until something is merged it basically has to be on all PRs.

**Review by meagancheese — APPROVED** [review]
One nit, but otherwise LGTM.

File: src/main/java/com/google/sps/data/DatastoreScheduledInterviewDao.java [hunk 2]
```diff
@@ -156,13 +157,19 @@ public Entity scheduledInterviewToEntityForUpdate(ScheduledInterview scheduledIn
...
+            "startTime", FilterOperator.GREATER_THAN_OR_EQUAL, minTime.toEpochMilli());
+    // Queries can only perform inequality filters on one parameter, and so instead
+    // of using endTime for the maxFilter, startTime is used and the maxTime has 60
+    // minutes subtracted from it to be equal to the latest possible startTime.
     Filter maxFilter =
-        new FilterPredicate("startTime", FilterOperator.LESS_THAN_OR_EQUAL, (maxTime - 3600000));
+        new FilterPredicate(
+            "startTime",
+            FilterOperator.LESS_THAN_OR_EQUAL,
+            maxTime.minus(60, ChronoUnit.MINUTES).toEpochMilli());
```
Comment (apluscs) [inline src/main/java/com/google/sps/data/DatastoreScheduledInterviewDao.java:176]: Add a comment to this function that states `maxTime` needs to be the first time after the desired range and with an example. Like if we wanted interviews starting >= 2:00PM and ending <= 6:00PM on 6/6/2020 then `maxTime` needs to be 6:00PM on that day.

**Review by apluscs — APPROVED** [review]
One comment request

**Review by acmcarther — APPROVED** [review]

**Pull request merged**